cloudevents/SDK.md

```diff
@@ -141,3 +141,64 @@ Each SDK must provide examples using at least HTTP transport of:
+||
+| **[v0.3](https://github.com/cloudevents/spec/tree/v0.3)** |
+| [CloudEvents Core](https://github.com/cloudevents/spec/blob/v0.3/spec.md)                                                                                  | :x: | :heavy_check_mark: | :heavy_check_mark:  | :heavy_check_mark: | :x: | :x: | :heavy_check_mark:    | :heavy_check_mark:  | :heavy_check_mark:  |
+| |
+| [AMQP Event Format](https://github.com/cloudevents/spec/blob/v0.3/amqp-format.md)                                                                          | :x: |     | :x:  | :x: |     |     |        |      | :x:  |
+| [JSON Event Format](https://github.com/cloudevents/spec/blob/v0.3/json-format.md)                                                                          | :x: |     | :heavy_check_mark:  | :heavy_check_mark: |     |     |        |      | :heavy_check_mark:  |
+| [Protobuf Event Format](https://github.com/cloudevents/spec/blob/v0.3/protobuf-format.md)                                                                  | :x: |     | :heavy_check_mark:  | :x: |     |     |        |      | :x:  |
+||
+| [AMQP Binding](https://github.com/cloudevents/spec/blob/v0.3/amqp-transport-binding.md)                                                                    | :x: |     | :heavy_check_mark:  | :x: |     |     |        |      | :x:  |
+| [HTTP Binding](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md)                                                                    | :x: |     | :heavy_check_mark:  | :heavy_check_mark: |     |     |        |      | :heavy_check_mark:  |
+| [HTTP Content Mode Binary](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md)                                                        | :x: |     | :heavy_check_mark:  | :heavy_check_mark: |     |     |        |      | :heavy_check_mark:  |
+| [HTTP Content Mode Structured](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md)                                                    | :x: |     | :heavy_check_mark:  | :heavy_check_mark: |     |     |        |      | :heavy_check_mark:  |
+| [HTTP Content Mode Batch](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md)                                                         | :x: |     | :x:  | :heavy_check_mark: |     |     |        |      | :heavy_check_mark:  |
+| [Kafka Content Mode Binary](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/bindings/kafka-protocol-binding.md#32-binary-content-mode)         | :x: |     | :heavy_check_mark:  | :heavy_check_mark: |     |     |        |      | :heavy_check_mark:  |
+| [Kafka Content Mode Structured](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/bindings/kafka-protocol-binding.md#33-structured-content-mode) | :x: |     | :heavy_check_mark:  | :heavy_check_mark: |     |     |        |      | :heavy_check_mark:  |
```

Comment:
Missing "Kafka Binding" raw for 0.3 and the links point to v1.0.2 spec

Reply:
fixed